plugins/BEdita/Core/src/Utility/Plugins.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2017 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\Core\Utility;
+
+use Cake\Core\Configure;
+use Cake\Core\Plugin;
+
+/**
+ * Plugins utility class
+ *
+ */
+class Plugins
```

Comment:
I think that having `Plugin` and `Plugins` might lead to some confusion. I'd rather rename this class to `\BEdita\Core\Plugin` and let it extend `\Cake\Core\Plugin`. We could then use `\BEdita\Core\Plugin` everywhere with the added benefits of `loadFromConfig()` method.